examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Jobs/Using_PennyLane_with_Braket_Jobs.ipynb

```diff
@@ -107,14 +107,25 @@
     "nx.write_adjlist(graph, input_file_path)"
```

Comment:
<p>1) Max-Cut</p><p> </p><p>2) Braket devices, <strong>the </strong><code>parametrize_differentiable</code> keyword </p><p> </p><p>3) The <a href="https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs.html" target="_blank">Amazon Braket developer guide</a> contains additional details about parametric circuits.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>